src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Utilities/BaseTest.cs

```diff
@@ -229,14 +229,31 @@ protected static string RunProcess (string exe, string args)
 				CreateNoWindow = true,
 				WindowStyle = ProcessWindowStyle.Hidden,
 			};
-			using (var proc = Process.Start (info)) {
+			using (var proc = new Process ()) {
+				StringBuilder standardOutput = new StringBuilder (), errorOutput = new StringBuilder ();
+				proc.StartInfo = info;
+				proc.OutputDataReceived += new DataReceivedEventHandler ((sender, e) => {
+					if (!string.IsNullOrEmpty (e.Data))
+						standardOutput.Append (Environment.NewLine + e.Data);
```

Comment:
Why place `Environment.NewLine` *first*?  This will introduce an "empty line" at the *beginning* of output, which might look "weird".

This also results in extra string temporaries.  This would likely be "nicer" to the GC:

```csharp
standardOutput.Append (e.Data).Append (Environment.NewLine);
```

Reply:
There is also an `AppendLine` method:

https://docs.microsoft.com/en-us/dotnet/api/system.text.stringbuilder.appendline?view=netframework-4.7.2

```suggestion
						standardOutput.AppendLine (e.Data);
```